.gitignore

```diff
@@ -27,3 +27,6 @@ pnpm-debug.log*
 *.njsproj
 *.sln
 *.sw?
+
+# Internal Notes
+MYREADME.md
```

Comment:
```suggestion
```
you can make a .gitignore on your machine that won't affect the project here. just put it one level higher than the project.